### PR TITLE
Browse Tests

### PR DESCRIPTION
**Pull request by alesgenova**

### CODE REVIEW

File: girder/cluster_filesystem/plugin_tests/constants.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    'group': 'group',
+    'user': 'user',
+    'mode': stat.S_IFREG,
+    'date': 'Feb  7 16:26',
+    'size': 78910
+}
+
+FILE_ID_FIELDS = {
+    'clusterId': 'dummy',
+    'path': "{}/{}".format(PATH, FILE1['name'])
```
Comment (cjh1) [inline girder/cluster_filesystem/plugin_tests/constants.py]: Can we use %s string formatting here.

File: girder/cluster_filesystem/plugin_tests/newt_test.py [hunk 2]
```diff
@@ -23,141 +23,230 @@
...
+
+def _assert_dir(listed, received, id_fields, is_curr=False):
+    _assert_base(listed, received, id_fields, is_curr)
+    assert not received['public']
+    assert received['parentCollection'] == 'folder'
+    
+    listed_parent = id_fields.copy()
+    received_parent = json.loads(urllib.parse.unquote_plus(received['parentId']))
+    _assert_parent(listed_parent, received_parent, is_curr)
+    #assert id['path'] == os.path.join(parent_id['path'], received['name'])
```
Comment (cjh1) [inline girder/cluster_filesystem/plugin_tests/newt_test.py]: Can we delete if we no longer need it?

File: girder/cluster_filesystem/plugin_tests/newt_test.py [hunk 3]
```diff
@@ -23,141 +23,230 @@
...
+
+    assert received['_modelType'] == 'item'
+
+def _assert_file(listed, received, id_fields):
+    _assert_base(listed, received, id_fields)
+    assert received['_modelType'] == 'file'
+
+
+
+
```
Comment (cjh1) [inline girder/cluster_filesystem/plugin_tests/newt_test.py]: Can we get rid of this space.

File: girder/cluster_filesystem/plugin_tests/newt_test.py [hunk 4]
```diff
@@ -23,141 +23,230 @@
...
+    received_parent = json.loads(urllib.parse.unquote_plus(received['parentId']))
+    _assert_parent(listed_parent, received_parent, is_curr)
+    #assert id['path'] == os.path.join(parent_id['path'], received['name'])
+    assert received['_modelType'] == 'folder'
+
+
+def _assert_item(listed, received, id_fields, is_curr=False):
+    _assert_base(listed, received, id_fields)
+
+    listed_parent = id_fields.copy()
```
Comment (cjh1) [inline girder/cluster_filesystem/plugin_tests/newt_test.py]: Why this copy?

File: girder/cluster_filesystem/server/dateutils.py [hunk 5]
```diff
@@ -0,0 +1,47 @@
+import datetime as dt
+
+def dateParser(timestring):
```
Comment (cjh1) [inline girder/cluster_filesystem/server/dateutils.py]: Snake case please.

File: girder/cluster_filesystem/plugin_tests/constants.py (hunk 1 above)
Comment (alesgenova) [reply girder/cluster_filesystem/plugin_tests/constants.py]: We could, but why? %s is the old formatting style, and is a little less flexible than .format().
See https://pyformat.info/

Comment (cjh1) [reply girder/cluster_filesystem/plugin_tests/constants.py]: We are using % style in the rest of the code base, so for better or worse I would favor consistency.

**Comment by psavery** [comment]
@cjh1 Now that we know that this works (for the most part), we can re-run travis-ci and possibly merge this branch. 

I don't have privileges to do either.

**Pull request closed without merging**